aws-iot-mitigationaction/src/test/java/com/amazonaws/iot/mitigationaction/ListHandlerTest.java

```diff
@@ -0,0 +1,126 @@
+package com.amazonaws.iot.mitigationaction;
+
+import static com.amazonaws.iot.mitigationaction.TestConstants.ACTION_ARN2;
+import static com.amazonaws.iot.mitigationaction.TestConstants.CREATION_DATE1;
+import static com.amazonaws.iot.mitigationaction.TestConstants.CREATION_DATE2;
+import static com.amazonaws.iot.mitigationaction.TestConstants.MITIGATION_ACTION_NAME2;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.when;
+
+import java.sql.Date;
```

Comment:
Looks like an accidental import. Let's run "organize imports" for all files that need it.

Reply:
Done